examples/website/geospatial/app.tsx

```diff
@@ -111,6 +113,30 @@ export default function App(props: AppProps) {
     loadedTable: null,
     error: null
   });
+  const stateRef = useRef<string>();
+  stateRef.current = state;
+
+  useEffect(() => {
+    if (isParquetReady) {
+      return;
+    }
+    async function prepareParquet() {
+      await preloadCompressions({modules: {'zstd-codec': ZstdCodec}});
```

Comment:
This makes the app too complicated. We can't expect apps to add all this code and go through these steps.

I think we need to find a way to wait for the preloads inside parquetLoader.

Reply:
Done